crates/subspace-proof-of-time/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 //! Proof of time implementation.
 
-#![cfg_attr(target_arch = "x86_64", feature(stdarch_x86_avx512))]
+#![cfg_attr(target_arch = "x86_64", feature(avx512_target_feature, stdarch_x86_avx512))]
```

Comment:
There is more problems with code than this. And this will not be needed once Rust toolchain is upgraded.

In fact soon `stdarch_x86_avx512` will not be needed either: https://github.com/rust-lang/rust/issues/111137#issuecomment-2911088418